From f697c0c5e6a8cddee0bbc19c3ba08979f91a97e0 Mon Sep 17 00:00:00 2001
From: Petr Oros <poros@redhat.com>
Date: Wed, 17 Sep 2014 16:24:00 -0400
Subject: [redhat] kabi: Adding symbol lookup_bdev (fs/block_dev.c)

Message-id: <3cba2da073bac5da85719c39c5e17523dd3799a4.1410510773.git.poros@redhat.com>
Patchwork-id: 95962
O-Subject: [PATCH RHEL7.1 120/233] kabi: Adding symbol lookup_bdev (fs/block_dev.c)
Bugzilla: 1071098
RH-Acked-by: Jeff Moyer <jmoyer@redhat.com>

Bugzilla: 1071098
    https://bugzilla.redhat.com/show_bug.cgi?id=1071098

    Description:
    ============
    Lookup a struct block_device by name. To get the block_device structure and take reference on it.

Signed-off-by: Petr Oros <poros@redhat.com>
---
 create mode 100644 redhat/kabi/kabi-module/kabi_ppc64/lookup_bdev
 create mode 100644 redhat/kabi/kabi-module/kabi_s390x/lookup_bdev
 create mode 100644 redhat/kabi/kabi-module/kabi_x86_64/lookup_bdev

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/redhat/kabi/kabi-module/kabi_ppc64/lookup_bdev b/redhat/kabi/kabi-module/kabi_ppc64/lookup_bdev
new file mode 100644
index 0000000..efd77fd
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_ppc64/lookup_bdev
@@ -0,0 +1,2 @@
+#1-
+0x91b7d3a4 lookup_bdev vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_s390x/lookup_bdev b/redhat/kabi/kabi-module/kabi_s390x/lookup_bdev
new file mode 100644
index 0000000..8a2cde7
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_s390x/lookup_bdev
@@ -0,0 +1,2 @@
+#1-
+0x58954f74 lookup_bdev vmlinux EXPORT_SYMBOL
diff --git a/redhat/kabi/kabi-module/kabi_x86_64/lookup_bdev b/redhat/kabi/kabi-module/kabi_x86_64/lookup_bdev
new file mode 100644
index 0000000..842b9b6
--- /dev/null
+++ b/redhat/kabi/kabi-module/kabi_x86_64/lookup_bdev
@@ -0,0 +1,2 @@
+#1-
+0xc9468df0 lookup_bdev vmlinux EXPORT_SYMBOL
-- 
1.7.1